projects/plugins/jetpack/_inc/connect-button.js

```diff
@@ -83,10 +76,10 @@ jQuery( document ).ready( function ( $ ) {
 					registration_nonce: jpConnect.registrationNonce,
 					_wpnonce: jpConnect.apiNonce,
 					from: connectButtonFrom,
-					no_iframe: 'original' === jpConnect.forceVariation,
+					no_iframe: true,
```

Comment:
Can't we also remove this line?

Reply:
I'll give it a shot. Left it there in case wpcom was expecting this present in some capacity, but nothing comes up in a global search. Will remove and see what happens.

Reply:
pushed, seems to be testing ok.